test/shell-pipeline-tests/openvino-tensorflow-housing/pipelines-test-openvino-tensorflow-housing.sh

```diff
@@ -35,13 +35,7 @@ oc apply -k "$AIEDGE_E2E_PIPELINE_DIR_PATH"/
 
 ## prepare parameters
 AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH="$AIEDGE_E2E_PIPELINE_DIR_PATH"/aiedge-e2e.pipelinerun-overridden.yaml
-cp "$AIEDGE_E2E_PIPELINE_DIR_PATH"/aiedge-e2e.pipelinerun.yaml "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
-sed -i "s|value: rhoai-edge-models|value: rhoai-edge-models-ci|" "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
-sed -i "s|value: bike-rentals-auto-ml|value: tensorflow-housing|" "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
-sed -i "s|Containerfile.seldonio.mlserver.mlflow|Containerfile.openvino.mlserver.mlflow|" "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
-sed -i "s|value: pipelines/models/|value: \"\"|" "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
-sed -i "s|\"invocations\"|\"v1/models/tensorflow-housing/versions/1:predict\"|" "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
-sed -i "s|name: bike-rentals-test-data|name: tensorflow-housing-test-data|" "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
+cp "$AIEDGE_E2E_PIPELINE_DIR_PATH"/aiedge-e2e.tensorflow-housing.pipelinerun.yaml "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
```

Comment:
```suggestion
cp "$AIEDGE_E2E_PIPELINE_DIR_PATH"/aiedge-e2e.tensorflow-housing.pipelinerun.yaml "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
sed -i "s|value: rhoai-edge-models|value: rhoai-edge-models-ci|" "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
sed -i "s|value: pipelines/models/|value: \"\"|" "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
```
+ possibly git to s3 or you can do it in the pipelinerun directly.

Reply:
As mentioned in the other comment (which I think is related to this, right?), I think having one test use a model from S3 and the other one use an example from Git would be better than having both from S3.

Reply:
@grdryn Yes, possibly. When I was implementing these tests I didn't see into it that much. The question is whether we need to test both example models on S3 and Git or not. Can it happen that one example model will run fine when fetched from S3 but not from Git or vice versa?

One thing which may theoretically occur is that we somehow move model/rename it in this Git repo and we forget to copy the changes to the S3 or vice versa as S3 contains a copy of these models. However, this wouldn't be an issue of our pipeline, of course. I am just thinking out loud whether we want to have the full coverage of all combinations or not.

If we are fine with having one for S3 and the other for Git then you can set relative path to `""` for the s3 seldonio-bike-rentals and remove that sed line from the test.

Reply:
> The question is whether we need to test both example models on S3 and Git or not. Can it happen that one example model will run fine when fetched from S3 but not from Git or vice versa?

In my opinion, the example models aren't the crucial thing that we want to verify the correctness of, it's the Pipeline and its Tasks. I don't think we would get any additional value from testing each model with each download method. By running one instance of the pipeline (for any model) with the S3 method, and another instance of the pipeline (for any model) with the Git method, then we're getting good coverage of the different Pipeline variations.
It's also good that the two examples we're using in the tests use different model servers, which tests the different Containerfiles that we have for those.

> One thing which may theoretically occur is that we somehow move model/rename it in this Git repo and we forget to copy the changes to the S3 or vice versa as S3 contains a copy of these models. However, this wouldn't be an issue of our pipeline, of course. I am just thinking out loud whether we want to have the full coverage of all combinations or not.

At that point, I think it's almost like what we'd be testing is the structure of the `rhoai-edge-models-ci` S3 bucket (which isn't an output of this project), rather than the pipeline or the example, right? We don't want to "overfit the data" (to borrow a term from machine learning) – as long as users make a PipelineRun that is correct for their model and where it's stored, our Pipeline should succeed. It shouldn't need to be completely like the structure or naming of our examples, as long as they can fit the Pipeline to it by configuring the parameters that we've made available.

> If we are fine with having one for S3 and the other for Git then you can set relative path to "" for the s3 seldonio-bike-rentals and remove that sed line from the test.

Done now, thanks! :+1:

Reply:
Yes, you are right, I think this way it's both sufficient and more efficient. If it was a very short test it would be doable but it doesn't make sense to test every possible combination with such long-running tests.